### ISSUE TITLE
connection update to websocket

### ISSUE THREAD

**Issue by madsheep** [boazsegev/iodine#20]
hi,

I'm trying to update from 0.3.6 to 0.4.6. Any guides on how to do that? Our application uses WebSockets heavily. We used to upgrade our connection like so: 
```
if env['upgrade.websocket?'.freeze] && env["HTTP_UPGRADE".freeze] =~ /websocket/i.freeze
  env['iodine.websocket'.freeze] = WebsocketHandler.new 
  [100, {"Sec-WebSocket-Protocol" => env["HTTP_SEC_WEBSOCKET_PROTOCOL"]}, []]
end
```
This was then translated by iodine to 101 `Switching Protocols` and iodine would add all the required headers for websocket update on it's own. 

Sadly this doesn't work anymore, as browser reports 'unexpected response code 100'. 

Examples in the readme say now I should use status 0 for the update? This, however, causes Chrome not to recognize the connection correctly anymore, and it seems iodine returns 500 error. I can set the status to 101 and do the update manually but that will require me to set all the other websocket protocol headers as well (Update, Sec-WebSocket-Accept etc).

Let me know what did I miss :)

**Comment by boazsegev** [comment]
Hi @madsheep ,

Thank you very much for asking and opening the issue. 

I couldn't figure out why my applications were working and your code was failing when I tested it.

Than I noticed the code was using `iodine.websocket` instead of `upgrade.websocket`.

As discussions about standardization for native Websocket support progressed, iodine conformed the upgrade process to [the proposed Rack Websocket Specs](https://github.com/boazsegev/iodine/blob/master/SPEC-Websocket-Draft.md#upgrading)

The `iodine` prefix was replaced with a generic one and backwards support ended on version 4.

Try:

```ruby
if env['upgrade.websocket?'.freeze] && env["HTTP_UPGRADE".freeze] =~ /websocket/i.freeze
  env['upgrade.websocket'.freeze] = WebsocketHandler.new 
  [0, {"Sec-WebSocket-Protocol" => env["HTTP_SEC_WEBSOCKET_PROTOCOL"]}, []]
end
```

Let me know if this helps, and good luck 👍🏻
B.
